e2e/public-web/test_error_pages.py

```
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
```

Ruff (0.14.7)

Shebang is present but file is not executable

(EXE001)